### PR TITLE
Homework 1 and 2

### PR DESCRIPTION
**Pull request by dncnwy**
Added Homework 1 under the main method in program.cs.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4847446&bid=16088089)

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,45 @@ class Program
...
+                Console.WriteLine("How much do you weight in pounds?");
+                weight = int.Parse(Console.ReadLine());
+                
+                float calculateBMI = 703 * weight / (height * height);
+
+                Console.WriteLine($"{name} is {age} years old, weighs {weight} pounds and is {height} inches tall. \nWith a weight of {weight} pounds and height of {height} inches, {name} has a BMI of {calculateBMI}.");
+
+                
+                Console.WriteLine("\nWould you like the check the BMI of someone else? (y/n)");
+                askAgain = Console.ReadLine().ToLower();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4847446)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,45 @@ class Program
...
+                Console.WriteLine("What is your name?");
+                name = Console.ReadLine();
+                Console.WriteLine("How old are you?");
+                age = int.Parse(Console.ReadLine());
+                Console.WriteLine("How tall are you in inches?");
+                height = int.Parse(Console.ReadLine());
+                Console.WriteLine("How much do you weight in pounds?");
+                weight = int.Parse(Console.ReadLine());
+                
+                float calculateBMI = 703 * weight / (height * height);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Cast one of the operands of this division to 'float'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4847446)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,45 @@ class Program
...
+
+                Console.WriteLine($"{name} is {age} years old, weighs {weight} pounds and is {height} inches tall. \nWith a weight of {weight} pounds and height of {height} inches, {name} has a BMI of {calculateBMI}.");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Split this 219 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4847446)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4847446&bid=16088089)

**Comment by dncnwy** [comment]
I didn't have an option to add the homework label.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -1,12 +1,52 @@
...
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            string askAgain;
+            int age;
+            int weight;
+            int height;
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: You want to declare those variables as late as possible. This means at the very moment of first usage.
Also having the type float for weight and height is probably better.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4847446&bid=16088089)

**Pull request closed without merging**